packages/core/src/domain/configuration/configuration.ts

```diff
@@ -125,11 +130,18 @@ export function validateAndBuildConfiguration(initConfiguration: InitConfigurati
     )
   }
 
+  // Build Session Store options
+  const sessionStoreOptions: SessionStoreOptions = {
+    cookie: buildCookieOptions(initConfiguration),
+    allowFallbackToLocalStorage: !!initConfiguration.allowFallbackToLocalStorage,
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏since `allowFallbackToLocalStorage` is only used in the current context to determine strategy type, what about removing it from the store options?